Leave/leave_interface.py

```diff
@@ -154,6 +154,23 @@ def GetReasonOfLeaves(leaves_array):
 
     return leaves_array[0]["reason"]
 
+def GetMissingMembersByRole(members_all, role, voice_channel_members):
+
+    date = datetime.datetime.strptime(datetime.datetime.today().strftime('%d/%m/%Y'), '%d/%m/%Y')
+
+    role_member_list = list(filter(lambda member: role in member.roles, members_all))
+    role_member_in_voicechannel_list = list(filter(lambda member : role in member.roles, voice_channel_members))
+    not_here_all = list(set(role_member_list) - set(role_member_in_voicechannel_list))
```

Comment:
What do you mean by `all`?

Reply:
This was meant to mean both approved and unapproved. Reworded the variable name.